src/GraphQL/Resolvers/EventStreamResolver.cs

```diff
@@ -1,38 +1,40 @@
-using GraphQL.Subscription;
-
 namespace GraphQL.Resolvers
 {
-    public class EventStreamResolver<T> : IEventStreamResolver<T>
+    public class EventStreamResolver<T> : IEventStreamResolver
```

Comment:
T -> TReturnType to simplify compare with `EventStreamResolver<TSourceType, TReturnType>`